test/matchmaker/player.test.data.js

```diff
@@ -1,97 +1,97 @@
 const inputPlayers = [
   {
-    type: 'single',
```

Comment:
use single quotes for text

Reply:
Felix let a script run to replace all single quotes with double quotes :D

Reply:
@Adentalis check if you have the "prettier" plugin installed and the option "format on save" activated, it should do this automatically :) fixed it for now